Makefile

```diff
@@ -1,17 +1,27 @@
-all: 
-	@docker-compose -f ./srcs/docker-compose.yml up
+.DEFAULT_GOAL := help
```

Comment:
`DEFAULT_GOAL` sets the default goal/target when no is given via the commandline, i.e. as you say when only executing:

```
make
```

if no `DEFAULT_GOAL` is defined it takes the first goal/target from the Makefile.

Here it will default to `help`.